src/pages/settings/Profile/PersonalDetails/CountrySelectionPage.tsx

```diff
@@ -1,45 +1,30 @@
-import lodashGet from 'lodash/get';
-import PropTypes from 'prop-types';
+import type {StackScreenProps} from '@react-navigation/stack';
 import React, {useCallback, useMemo, useState} from 'react';
-import _ from 'underscore';
 import HeaderWithBackButton from '@components/HeaderWithBackButton';
 import ScreenWrapper from '@components/ScreenWrapper';
 import SelectionList from '@components/SelectionList';
 import useLocalize from '@hooks/useLocalize';
 import Navigation from '@libs/Navigation/Navigation';
+import type {SettingsNavigatorParamList} from '@libs/Navigation/types';
+import type {CountryData} from '@libs/searchCountryOptions';
 import searchCountryOptions from '@libs/searchCountryOptions';
 import StringUtils from '@libs/StringUtils';
 import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+import type {Route} from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
 
-const propTypes = {
-    /** Route from navigation */
-    route: PropTypes.shape({
-        /** Params from the route */
-        params: PropTypes.shape({
-            /** Currently selected country */
-            country: PropTypes.string,
+type CountrySelectionPageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.SETTINGS.PROFILE.ADDRESS_COUNTRY>;
 
-            /** Route to navigate back after selecting a currency */
-            backTo: PropTypes.string,
-        }),
-    }).isRequired,
-
-    /** Navigation from react-navigation */
-    navigation: PropTypes.shape({
-        /** getState function retrieves the current navigation state from react-navigation's navigation property */
-        getState: PropTypes.func.isRequired,
-    }).isRequired,
-};
-
-function CountrySelectionPage({route, navigation}) {
+function CountrySelectionPage({route, navigation}: CountrySelectionPageProps) {
     const [searchValue, setSearchValue] = useState('');
     const {translate} = useLocalize();
-    const currentCountry = lodashGet(route, 'params.country');
+    const currentCountry = route.params.country;
```

Comment:
Shouldn't this be `route.params?.country`? This is inconsistent with https://github.com/Expensify/App/pull/35307/files#diff-02dc5171dfb0335e880ff4994705c5607f8f7a82de55bd1d4fa241c499d9232dR47

Reply:
Actually I think params are always there 
<img width="391" alt="image" src="https://github.com/Expensify/App/assets/25230417/99565096-8278-41e0-87ee-97ccfbc2b9e4">
So this should be fine to left it as it is